package.json

```diff
@@ -30,6 +30,7 @@
     "scripts": {
         "build": "parcel build",
         "fix": "yarn eslint . --ignore-path .gitignore --ext .js,.jsx,.ts,.tsx --fix",
+        "postinstall": "andromatic-install -p platform-tools 'build-tools;33.0.2'",
```

Comment:
Is the cli available from the dependencies directly? I guess I don't really understand these multiple targets…

Reply:
Yes. Andromatic can be used both as a library (through the `main` property in the `package.json`) and as a CLI (through the `bin`) property. The targets are just for Parcel.